nautobot/core/api/views.py

```diff
@@ -721,14 +721,13 @@ class GetMenuAPIView(NautobotAPIVersionMixin, APIView):
 
     permission_classes = [IsAuthenticated]
 
-    def format_and_remove_hidden_menu(self, request, data, hide_restricted_ui):
+    def format_and_remove_hidden_menu(self, request, data, hide_restricted_ui=False):
```

Comment:
Let's just remove the `hide_restricted_ui` parameter altogether - this isn't a documented/guaranteed-stable API as far as I know.